pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
@fabpot Isn't there a way to make sure there is no wrong key in the recipe file? Here, we have a hidden issue: "alias" is not available but the bot doesn't care. Maybe Symfony/Config can be used to validate any recipe file and make sure there is no trailing or unintended data in the file :)